duckduckgo/iOS
# Brad/hd apptp rich notifs

Task/Issue URL: https://app.asana.com/0/42792087274227/1204417067280974/f
Tech Design URL:
CC: @samsymons 

**Description**:
This PR adds daily summary notifications for AppTP. Opening as a draft for now to facilitate review and optimization.



**Steps to test this PR**:
1. On the AppTP page enable notifications and set a desired fire time. Ensure notification happens at scheduled time.
2. Use the temporary "Trigger notification" button to instantly deliver a notification to see how the view looks.



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204551818281628
